com.alibabainc.dingtalk.yaml

```
@@ -0,0 +1,50 @@
+id: com.alibabainc.dingtalk
+runtime: org.freedesktop.Platform
+runtime-version: '24.08'
+sdk: org.freedesktop.Sdk
+command: Elevator.sh
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=x11
+  - --socket=pulseaudio
+  - --talk-name=org.freedesktop.FileManager1
+  - --talk-name=org.freedesktop.Notifications
+  - --talk-name=org.kde.StatusNotifierWatcher
+  - --device=all
+  - --filesystem=xdg-download
+
+modules:
+  - name: libxcrypt
+    buildsystem: autotools
+    config-opts: ["--enable-obsolete-api=glibc"]
```

```suggestion
    config-opts:
      - --enable-obsolete-api=glibc
```

I you want to use the JSON syntax make the manifest a JSON instead.